.github/workflows/pages.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
```

Comment:
The process implemented in this workflow is not documented in any way.

Additionally, there's no way to run this outside GitHub Actions (unless you use more third-party tools, such as Act).

How can we evaluate the result of a change before actually merging a PR?

Reply:
Done